ui/src/app/pages/admin/user/deprecated-admin-user.tsx

```diff
@@ -130,7 +130,7 @@ interface State {
   verifiedInstitutionOptions: Array<PublicInstitutionDetails>;
 }
 
-export const AdminUser = withRouter(
+export const DeprecatedAdminUser = withRouter(
```

Comment:
to be removed soon: https://precisionmedicineinitiative.atlassian.net/browse/RW-8235